torch/jit/_script.py

```diff
@@ -36,6 +36,14 @@
 from torch.overrides import (
     has_torch_function, has_torch_function_unary, has_torch_function_variadic)
 
+from torch.jit._monkeytype_config import (
+    monkeytype_trace,
+    JitTypeTraceConfig ,
+    JitTypeTraceStore
+)  # type: ignore[import]
+
+type_trace_db = JitTypeTraceStore()  # DB to hold all call traces from MonkeyType
```

Comment:
Since each call to `_script_pdt` creates a new instance, we probably don't need to initialize here.

Reply:
When monkeytype is not installed, then we need this to be initialized to be used in `torch/jit/frontend.py`